### DISCUSSION TITLE
Sponsorship is confusing

### DISCUSSION THREAD

**Discussion by jemail99** [d4rken-org/sdmaid-se#398] [Q&A] [answered]
Monthly sponsorship of $5 or over gives the reward of sponsorship which I assume unlocks the app cleaner and other "Pro" features.
The one time sponsorship donation states that it gives no rewards. Does this mean that a one time sponsorship donation does not unlock the app cleaner and "Pro" features?

**Comment by d4rken** [comment]
> The one time sponsorship donation states that it gives no rewards.

Can you elaborate on this?

The explanation text contains the phrase "same features, just different pricing", maybe there is a translation error in your locale?

**Comment by jemail99** [comment]
This is the text displayed for a monthly $5 sponsorship:
 ```
$5 a month

Become a Sponsor

You'll be @d4rken's 10th sponsor, helping them reach their goal of 50 monthly sponsors.
```

This is the text displayed for a monthly $10 sponsorship:
 ```
$10 a month
Custom amount.

You'll receive rewards listed in the $5 a month tier:

Become a Sponsor

You'll be @d4rken's 10th sponsor, helping them reach their goal of 50 monthly sponsors.
```

This is the text displayed for a one time $10 sponsorship here on github:
 ```
Selected tier
$10 one time
Custom amount. There are **no rewards associated with this sponsorship**.
You will count as a sponsor of @d4rken for 30 days. 
```
So, when one clicks the upgrade button in the github app version of SDMaid SE I am led to believe that sponsorship or becoming a sponsor will provide the benefit of unlocking the Pro features of SDMaid SE. When I arrive in github to select a level of sponsorship I am meet with the above choices. It is not at all clear what sponsorship is providing, there is no mention of unlocking the Pro features of SDMaid SE in return for sponsorship.
Hence my question, what exactly does a one time donation do with regards to unlocking the Pro features of SDMaid SE? Indeed what does any sponsorship do with regards to unlocking the Pro features of SDMaid SE? Is the unlocking limited to the duration of sponsorship only?

**Reply by d4rken** [reply, marked as answer]
Ah I misunderstood you, sorry.

> Hence my question, what exactly does a one time donation do with regards to unlocking the Pro features of SDMaid SE? Indeed what does any sponsorship do with regards to unlocking the Pro features of SDMaid SE? Is the unlocking limited to the duration of sponsorship only?

There is no technical connection. The "Pro Upgrade" in the FOSS version of SD Maid SE is not enforced. It's based on a honor system. I hope that the user does sponsor development, but if the user just closes the browser and returns to the app, the extra features are still enabled.
